projects/igniteui-angular/src/lib/accordion/accordion.component.spec.ts

```diff
@@ -232,6 +232,7 @@ describe('Rendering Tests', () => {
                 argsIng = { event: expArgs.event, cancel: expArgs.cancel, owner: accordion, panel: expArgs.owner };
             });
             accordion.panels[0].collapse();
+            accordion.panels[0].collapse();
```

Comment:
Why are we calling `collapse` on the first panel twice?

Reply:
I am calling collapse method twice, in order to check if panelCollapsing is emitted again after the panel already started collapsing.